WordPress/Classes/ViewRelated/Reader/FollowedSitesViewController.m

```diff
@@ -143,7 +143,7 @@ - (NSFetchRequest *)fetchRequest
 
 - (void)configureCell:(UITableViewCell *)cell atIndexPath:(NSIndexPath *)indexPath
 {
-    UIImage *defaultImage = [UIImage imageNamed:@"icon-feed"];
+    UIImage *defaultImage = [UIImage imageNamed:@"post-blavatar-placeholder"];
```

Comment:
Thanks -- I forgot to include a before and after of this in the original post:

![feedicons](https://cloud.githubusercontent.com/assets/1396552/12431325/e8644654-bec2-11e5-994f-fdb3ec1645fd.png)